config/default/manager_auth_proxy_patch.yaml

```diff
@@ -23,5 +23,6 @@ spec:
           name: https
       - name: manager
         args:
-        - "--metrics-addr=127.0.0.1:8080"
-        - "--enable-leader-election"
+        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--leader-elect"
+        - "--health-probe-bind-address=:8081"
```

Comment:
The manager.yaml and main didn't have added healthz / liveness endpoints, yet this defines a port for them.

Reply:
Same as above, following the upgrade steps. Do you want me to remove the `- "--health-probe-bind-address=:8081"` line?

Reply:
No, you could add the health checks instead as I commented below.

Reply:
done